### PR TITLE
[DOCS-97] Set up dividing line / spacing between doc content and feedback links

### PR DESCRIPTION
**Pull request by mjang-cobalt**
Hi @dannyleong39 I hope this is an "easy" review for you. If you're busy, you're welcome to suggest a different designer

- Update, will assign to Maria T once she has GitHub access.

### CODE REVIEW

File: layouts/partials/feedback.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <div id="feedback">
-  <p>Was this page helpful?</p>
+<br>
+<hr size="8" width="100%" color="0047AB">
```
Comment (mjang-cobalt) [inline layouts/partials/feedback.html:3]: I could visualize increasing the size and width

Comment (mtsuji-cobalt) [inline layouts/partials/feedback.html:3]:
```suggestion
<hr size="10" width="100%" color="0047AB">  
```
```suggestion
<hr size="8" width="100%" color="0047AB">  
```

Comment (mtsuji-cobalt) [inline layouts/partials/feedback.html:3]:
```suggestion
<hr size="10" width="100%" color="0047AB">  
```
```suggestion
<hr size="8" width="100%" color="0047AB">  
```

File: layouts/partials/feedback.html [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <div id="feedback">
-  <p>Was this page helpful?</p>
+<br>
+<hr size="10" width="100%" color="0047AB">
```
Comment (mjang-cobalt) [inline layouts/partials/feedback.html:3]:
```suggestion
<hr size="8" width="100%" color="0047AB">  
```

Comment (mtsuji-cobalt) [inline layouts/partials/feedback.html:3]:
```suggestion
<hr size="10" width="120%" color="0047AB">  
```

**Comment by mjang-cobalt** [comment]
Thanks for your comments, @mtsuji-cobalt .

> i would advocate for a wider width to distinguish it from the grey lines that are used in a table format on some of the other pages (e.g. Severity Levels). if that ends up looking strange, we could also adjust the thickness/height of the line.

I tried it at 120%, and this is how the page appears. Note how the blue line overlaps the vertical gray line in the right side of the browser. IMO, we need some sort of line in the right side for the list of links to "Edit this Page" and section titles.

Screen Shot 2022-03-24 at 1 35 49 PM

From my tests, the lines cross, even at "102%" width. So I think it's best to stay at !00%.

> also, we'd have to remove the existing grey line right below 'Assuming you’ve received an email invitation, take the next step. Sign in to Cobalt.' on the Getting Started page and other relevant pages.

Fortunately, the other PR that you just reviewed removes that grey line. I've merged that PR, and the changes now appear in the Deploy Preview URL: https://deploy-preview-107--cobalt-docs.netlify.app/

If you're OK with this (including the 100% line width), I'd appreciate your approval of this PR.

**Review by mtsuji-cobalt — APPROVED** [review]
Looks good - I looked at all the pages and the line at 100% works all across.

**Comment by mjang-cobalt** [comment]
I'm hesitating for the moment, as the line _barely_ appears in Firefox. I'll merge as is -- unless I can find a "simple" solution (as I'm not a Front End developer)

**Pull request merged**